spaces/mstable/skin.scss

```
@@ -0,0 +1,10 @@
+.mstable {
+  --primary-color: ##1065E0;
```

You have 2 # in the hexadecimal color.
```suggestion
  --primary-color: #1065E0;
```